editoast/src/views/test_app.rs

```diff
@@ -39,6 +39,7 @@ pub(crate) struct TestAppBuilder {
     db_pool: Option<DbConnectionPoolV2>,
     core_client: Option<CoreClient>,
     osrdyne_client: Option<OsrdyneClient>,
+    disable_authorization: bool,
```

Comment:
```suggestion
    enable_authorization: bool,
```

I'd reverse the flag as we don't want to bother with authorization for most tests. But for those where we do, I think this reads more easily.

Reply:
Done.